### ISSUE TITLE
Add support for building on OS/X / homebrew

### ISSUE THREAD

**Issue by pilhuhn** [owntone/owntone-server#215]
One upon a time there was a macports build for mt-daapd available.
It would be cool if building via homebrew would work for forked-daapd.

I've started looking into it, but am currently fighting with/against gettext from homebrew vs the supplied one.

**Comment by couteau** [comment]
Attached is a diff file showing some work I did a while back on getting forked-daapd to compile under OS X.  It is against an older master and doesn't patch against the current master, but it shouldn't be hard to figure out what the changes are.  The biggest issue was the absence of high-resolution timers in OS X.  I wrote a set of wrappers to map those functions onto regular low-res timer functions, but I never got the whole thing compelling, so I wasn't able to test the impact of using low-res timers  -- I am assuming the place you'd see problems with it is in playback.  There were also some changes to use dnssd instead of avahi for zeroconf and some other tweaks to get the automake/configure scripts working.

[forked-daapd-osx.diff.zip](https://github.com/ejurgensen/forked-daapd/files/163664/forked-daapd-osx.diff.zip)
